seqr-gke-nodepool/main.tf

```diff
@@ -33,11 +33,19 @@ resource "google_container_node_pool" "node_pool" {
     }
   }
 
-  node_count     = var.node_pool_count
   node_locations = [var.node_location]
 
   upgrade_settings {
     max_surge       = "1"
     max_unavailable = "0"
   }
+
+  node_count         = (var.max_node_pool_count > 0) ? null : var.min_node_pool_count
```

Comment:
There's also `initial_node_count`, but the [docs](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_node_pool#initial_node_count) say not to use it if you don't need it as changing it triggers a re-spin.

Reply:
yes, i would avoid the initial_node_count. It's frustrating that most the examples on the docs page use it 🤔